charts/ingress/CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## Unreleased
+
+### Changes
+
+* Autogenerated name of `ValidatingWebhookConfiguration` changed from
+  `<FULLNAME>-validations` to `<FULLNAME>-<CHART-INSTALLATION-NAMESPACE>-validations`
+  to avoid collisions in case of multiple installations.
+  [#1211](https://github.com/Kong/charts/pull/1211)
```

Comment:
I'm not sure if we want to include this here. Historically (changelog entries below) we've only mentioned that we bump the `kong/kong` dependency and linked to the changelog of that subchart. WDYT?

Reply:
Yeah, I did it mechanicaly :)